distributed/diagnostics/tests/test_widgets.py

```diff
@@ -235,8 +232,7 @@ def test_progressbar_cancel(client):
 @gen_cluster()
 async def test_multibar_complete(s, a, b):
     s.update_graph(
-        tasks=valmap(
-            dumps_task,
```

Comment:
Wow.   These tests are ancient.  We stopped white_box unit tests like this a *long* time ago.